### ISSUE TITLE
The \"path\" argument must be of type string. Received undefined

### ISSUE THREAD

**Issue by ZiiMakc** [mcansh/remix-fastify#49]
On server start following error is thrown `The \"path\" argument must be of type string. Received undefined`

For server like:
```
  await app.register(remixFastifyPlugin, {
    build: path.join(__dirname, '../build/index.js'), // resolves fine
    mode: isDev ? 'development' : 'production',
  });
```

Error happens in this file:  `node_modules/@mcansh/remix-fastify/plugin.js`
```
let remixFastify = async (fastify, options = {}) => {
  let {build, mode = process.env.NODE_ENV} = options;
  (0, import_tiny_invariant.default)(build, "You must provide a build");

  console.error(build)  // resolves fine to D:\apps\xx\frontend\build\index.js
  console.error(typeof build === "string" ) // true
  console.error(resolvedBuild) // {}


  let resolvedBuild = typeof build === "string" ? require(build) : build;  // require(build) resolves to {}

  if (!fastify.hasContentTypeParser("*")) {
    fastify.addContentTypeParser("*", (_request, payload, done) => {
      done(null, payload);
    });
  }
  fastify.register(import_fastify_racing.default, {handleError: true});
  let ROOT_DIR = process.cwd();
  let PUBLIC_DIR = path.join(ROOT_DIR, "public");
  let ASSET_DIR = path.join(ROOT_DIR, resolvedBuild.assetsBuildDirectory); // resolvedBuild.assetsBuildDirectory is undefined - error
```

**Comment by mcansh** [comment]
is this in development?

**Comment by ZiiMakc** [comment]
> is this in development?
yes

**Comment by qianjiachun** [comment]
same

**Comment by mcansh** [comment]
potentially fixed by #76 and released in 2.7.0

if it is still broken, feel free to open a new issue
